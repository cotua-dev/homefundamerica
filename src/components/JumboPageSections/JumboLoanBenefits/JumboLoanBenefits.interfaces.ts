export interface FetchJumboLoanBenefitsImageQuery {
    jumboLoanBenefitsImage: {
        publicURL: string;
    };
}
