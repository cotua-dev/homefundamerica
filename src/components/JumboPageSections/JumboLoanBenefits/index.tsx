import React, {
    FunctionComponent,
    ReactElement,
} from 'react';
import {
    useStaticQuery,
    graphql,
} from 'gatsby';
import { FetchJumboLoanBenefitsImageQuery } from './JumboLoanBenefits.interfaces';
import { jumboLoanBenefitsSection } from './JumboLoanBenefits.module.scss';
import * as sharedStyles from '../../sharedStyles/shared.module.scss';

export const JumboLoanBenefits: FunctionComponent = (): ReactElement => {
    const queryResult = useStaticQuery<FetchJumboLoanBenefitsImageQuery>(graphql`
        query {
            jumboLoanBenefitsImage: file(name: {eq: "AdobeStock_267403891"}, extension: {eq: "jpg"}) {
                publicURL
            }
        }
    `);

    return (
        <section
            id={jumboLoanBenefitsSection}
            className={sharedStyles.loanNegative}
        >
            <div className={sharedStyles.contentWrapper}>
                <h2>{`Jumbo & Super Jumbo Loan Benefits`}</h2>
                <p>
                    {`
                        As a premier provider of million dollar home loans,
                        Manhattan Banc is expert at accommodating large loan
                        amounts and creating custom mortgage solutions:
                    `}
                </p>
                <ul>
                    <li>{`Loan amounts up to $30 million`}</li>
                    <li>{`Bank Statement Loans`}</li>
                    <li>{`Underwriting Flexibility`}</li>
                    <li>{`Portfolio Loan Options`}</li>
                    <li>{`Extensive range of mortgage programs and options`}</li>
                    <li>{`Interest-only options available`}</li>
                    <li>{`Unique properties considered`}</li>
                    <li>{`90% financing on purchase loans`}</li>
                    <li>{`Highly competitive non-resident alien programs`}</li>
                </ul>
            </div>
            <div className={sharedStyles.imageWrapper}>
                <img
                    src={queryResult?.jumboLoanBenefitsImage.publicURL}
                    alt="Jumbo loan benefits image"
                />
            </div>
        </section>
    );
}
