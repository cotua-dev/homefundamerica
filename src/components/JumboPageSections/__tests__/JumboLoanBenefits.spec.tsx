import React from 'react';
import {
    render,
    RenderResult,
} from '@testing-library/react';
import { JumboLoanBenefits } from '../JumboLoanBenefits';

describe('JumboLoanBenefits Component', (): void => {
    test('renders', (): void => {
        const { container }: RenderResult = render(<JumboLoanBenefits/>);
        expect(container.firstChild).toMatchSnapshot();
    });
});
