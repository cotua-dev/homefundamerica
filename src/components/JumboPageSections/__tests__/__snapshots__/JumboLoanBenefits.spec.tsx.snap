// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JumboLoanBenefits Component renders 1`] = `
<section
  id="jumboLoanBenefitsSection"
>
  <div>
    <h2>
      Jumbo & Super Jumbo Loan Benefits
    </h2>
    <p>
      
                        As a premier provider of million dollar home loans,
                        Manhattan Banc is expert at accommodating large loan
                        amounts and creating custom mortgage solutions:
                    
    </p>
    <ul>
      <li>
        Loan amounts up to $30 million
      </li>
      <li>
        Bank Statement Loans
      </li>
      <li>
        Underwriting Flexibility
      </li>
      <li>
        Portfolio Loan Options
      </li>
      <li>
        Extensive range of mortgage programs and options
      </li>
      <li>
        Interest-only options available
      </li>
      <li>
        Unique properties considered
      </li>
      <li>
        90% financing on purchase loans
      </li>
      <li>
        Highly competitive non-resident alien programs
      </li>
    </ul>
  </div>
  <div>
    <img
      alt="Jumbo loan benefits image"
    />
  </div>
</section>
`;
